scripts/check_blocking_packages.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import argparse
+
+import rosdistro
+import rosdistro.dependency_walker
+
+index = rosdistro.get_index(rosdistro.get_index_url())
+valid_distro_keys = index.distributions.keys()
+valid_distro_keys.sort()
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('--distro', metavar='distribution', type=str,
+                    help='The distribution to check packages for',
+                    default=valid_distro_keys[-1])
+
+# If not specified, check for all repositories in the previous distribution
+parser.add_argument('--repositories',
+                    metavar='repositories', type=str, nargs='*',
+                    help='Unreleased repositories to check dependencies for')
+
+parser.add_argument('--depth',
+                    metavar='depth', type=int,
+                    help='Maxmium depth to crawl the dependency tree',
+                    default=None)
+
+args = parser.parse_args()
+
+distribution_key = args.distro
+repository_names = args.repositories
+dependency_walker_depth=args.depth
```

Comment:
These variable name seem to be not shorter then their args field. Why add new variables for them?